src/components/Header/index.js

```diff
@@ -245,6 +249,10 @@ export default class Header extends Component {
 
           <SiteName />
 
+          {/* isOverflowing && (
+            <LoginCTA className={styles.headerLoginCta}>Connect</LoginCTA>
+          ) */}
```

Comment:
would you mind just cleaning this one up and removing the commented code?

Reply:
No problem, I can do that today.